frontend/package.json

```diff
@@ -286,12 +285,13 @@
     "cypress-jest-adapter": "^0.1.1",
     "cypress-multi-reporters": "^1.4.0",
     "enzyme": "3.10.x",
+    "esbuild-loader": "^4.2.2",
     "file-loader": "6.2.0",
     "find-up": "4.x",
-    "fork-ts-checker-webpack-plugin": "6.5.3",
+    "fork-ts-checker-webpack-plugin": "9.0.2",
```

Comment:
`esbuild-loader` does not cover TS type checking, so we will have to keep` fork-ts-checker`.

> esbuild does not type check your code. And according to the [esbuild FAQ](https://esbuild.github.io/faq/#:~:text=typescript%20type%20checking%20(just%20run%20tsc%20separately)), it will not be supported.